source_code/src/GUI/mini_gui_screen_functions.c

```diff
@@ -460,7 +464,13 @@ void guiDisplayRawString(uint8_t stringID)
 */
 void guiDisplayLoginOrPasswordOnScreen(char* text)
 {
+    #ifdef MINI_VERSION
+    miniOledAllowTextWritingYIncrement();
+    #endif
     guiDisplayTextInformationOnScreen(text);
+    #ifdef MINI_VERSION
+    miniOledPreventTextWritingYIncrement();
```

Comment:
nice one! but only because of https://github.com/limpkin/mooltipass/blob/master/source_code/src/OLEDMINI/oledmini.c#L991

which makes me think... why did I ever name miniOledMaxTextY and not miniOledMaxTextX?